proposal.html

```diff
@@ -14,30 +14,50 @@
     <div class="container">
         <div class="row">
             <div class="header">
-                header
+                <h3>Pwitter</h3>
+                <p>An website which will allow users to tweet similar to twitter</p>
             </div>
         </div>
         <div class="row">
             <div class="mid">
                 <div class="mid-box">
-                    mid-box-left
+                    RESTFul API?<br>
+                    How will this api give insight to users?<br>
+                    <br>
+                    Would collect: name, bio, email, location, website, and birth date<br>
+                    <br>
+                    USER -> fname, lname, password, dob, email, website, location, and bio<br>
                 </div>
                 <div class="mid-box">
-                    mid-box-right
+                    People who love to share their thoughts with others<br>
+                    <br>
+                    Goal 1: Have a user-friendly UI<br>
+                    Goal 2: Have a fully polished website by the end of the semester<br>
+                    <br>
+                    We selected this topic because we think this will really challenge us to
+                    the fullest and can keep adding complexities as we go.
                 </div>
             </div>
         </div>
         <div class="row">
             <div class="footer-container">
                 <div class="footer-items">
                     <div class="footer-box">
-                        footer-box-left
+                        - Link to API<br>
+                        - Twitter.com<br>
+                        <br>
+                        Resources:<br>
+                        - reactjs.org<br>
+                        - expressjs.com
                     </div>
                     <div class="footer-box">
-                        footer-box-left
+                        Questions:
                     </div>
                     <div class="footer-box">
-                        footer-box-left
+                        Accessibility implementation thoughts:<br>
+                        Maybe text to speech?<br>
+                        Light/Dark mode?<br>
+                        Font change button?<br>
```

Comment:
Can you add section headers here too?

- **Resource Links**:
- **Questions + Concerns**:
- **Accessibility Implementation**: